packages/core/src/components/links/links.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+	SpaceProps,
+	ColorProps,
+	TypographyProps,
+	LayoutProps,
+} from '../globals/globals';
+import { useClassNames } from '../../hooks/use-class-names';
+import styles from './links.module.scss';
+
+export type LinkProps = {
+	cfg?: SpaceProps & ColorProps & TypographyProps & LayoutProps;
+	className?: string;
+	underline?: boolean;
+	testId?: string;
+	[key: string]: any;
+};
+export const Link: React.FC<LinkProps> = ({
+	cfg: globalStyles,
+	underline = false,
+	className,
+	testId,
+	children,
+	...props
+}) => {
+	const classNames = useClassNames(globalStyles, [
+		styles.link,
+		{ [styles['link-underline']]: underline },
+		className,
+	]);
+
+	const anchorProps = {
+		'data-testid': testId,
+		className: classNames,
+		tabIndex: props.href ? null : 0,
+		href: props.href ? props.href : 'javascript:;',
```

Comment:
when lunching the docz app, I get the following warning in console:
`Warning: A future version of React will block javascript: URLs as a security precaution. Use event handlers instead if you can. If you need to generate unsafe HTML try using dangerouslySetInnerHTML instead. React was passed "javascript:;".`

Probably we could change it to be something like:
```
href: props.href ? props.href : '#',
onClick: props.href ? null : function(e) { e.preventDefault(); },
```

The call to `preventDefault()` is to prevent the viewport jumping to the top of the page.

Reply:
I didn't want to use # as the href target because it was causing the jump to top as you say.  I'll implement your suggestion and re-test the links to ensure that they work.

The presence of an href is required even for links with an onClick event to allow keyboard users to activate the anchor tags with the return key.

Reply:
@vrojastpr I have implemented as suggested with a slight variation just to ensure that any onclick handlers attached execute.